src/vs/workbench/services/preferences/common/preferencesValidation.ts

```diff
@@ -45,15 +48,19 @@ export function createValidator(prop: IConfigurationPropertySchema): (value: any
 		}
 
 		if (isNumeric) {
-			if (value === '' || isNaN(+value)) {
+			if (isNullValue(value) || isNaN(+value)) {
```

Comment:
I think the '' check is still needed because `+''` converts to 0 (thanks js)

Reply:
Oh I see that the function does that, but then I think it has the wrong name...

Reply:
isNullValue checks for ''. Maybe I can rename it to `isNullOrEmpty`

Reply:
We already have an `isUndefinedOrNull` helper